packages/core/src/execution-engine/__tests__/workflow-execute.test.ts

```diff
@@ -842,52 +842,6 @@ describe('WorkflowExecute', () => {
 			);
 		});
 
-		//                 ►►
-		//                ┌──────┐
-		//                │orphan│
-		//                └──────┘
-		//  ┌───────┐     ┌───────────┐
-		//  │trigger├────►│destination│
-		//  └───────┘     └───────────┘
-		test('works with a single node', async () => {
```

Comment:
This is not the responsibility of the partial execution flow anymore. This is handled as a `FullManualExecutionFromUnknownTriggerPayload` now.